tests/test.mjs

```diff
@@ -102,11 +116,67 @@ async function getFullReport(page) {
 }
 
 async function startRequestProfiling(page) {
+  const regex = /(.*)\/changeset\/([\w\d]+)(\/.*)?/;
   page.on("requestfinished", async (request) => {
     const url = request.url();
     if (!requests[url]) {
       requests[url] = [];
     }
     requests[url].push(request.timing());
+    if (needChangesetId) {
+      const match = url.match(regex);
+      if (match) {
+        const changesetId = match[2];
+        console.log(`Changeset id found: ${changesetId}`);
+      }
+    }
+
   });
 }
+
+export async function teardownBackend(requestParams, response, context, ee, next) {
+  const userCred = {
+    email: username,
+    password: password
+  }
+
+  const authClientConfig = {
+    clientId,
+    redirectUri,
+    scope,
+    authority
+  };
+
+  if (deleteBackend) {
+    console.log("Manually deleting provisioned backend...");
+    const client = new TestBrowserAuthorizationClient(authClientConfig, userCred);
+    const accessToken = await client.getAccessToken();
+
+    try {
+      const orchestratorUrl = `${orchestratorBaseUrl}/${backendName}/${backendVersion}/mode/1/context/${iTwinId}/imodel/${iModelId}/changeset/${changeSetId}/client/${authClientConfig.clientId}`;
+
+      const response = await got.delete(orchestratorUrl, {
```

Comment:
why got over fetch?

Reply:
no reason in particular, I copy-pasted from orchestrator's snippet